src/sfctl/helps/cluster_upgrade.py

```diff
@@ -165,6 +165,13 @@
           short-summary: The maximum allowed percentage of upgrade domain delta
             health degradation during the upgrade. Allowed values are integer
             values from zero to 100.
+        - name: --application-health-policies
+          type: string
+          short-summary: JSON encoded dictionary of pairs of application type
+            name and maximum percentage unhealthy before raising error
+    examples:
+        - name: Start a cluster configuration update
+          text: sfctl sa-cluster config-upgrade --cluster-config <YOUR CLUSTER CONFIG>
```

Comment:
What format is the value of `cluster config`? Can you include a concrete example? Same for the new param `application-health-policies`. What format should the JSON be?

 Can you also improve the help text for cluster config? "The cluster configuration" is not all that helpful :) Thanks!!

Reply:
`This is exactly what I was saying.. the config is huge and it doesn't make sense to put the full json blob here.` Can you find an example elsewhere online then? If so, maybe we should include a link to it, or a mention of it, so users have an easier time finding an example somewhere. I think that users might know what it is and have a rough idea, but having the exact syntax down is another thing